test/cat.jl

```diff
@@ -228,6 +228,47 @@ module TestCat
         @test [df1; df2] == df3
     end
 
+    @testset "vcat with :union" begin
+        df1 = DataFrame(A = 1:3, B = 4:6)
+        df2 = DataFrame(A = 7:9)
+        df3 = DataFrame(B = 4:6, A = 1:3)
+
+        @test vcat(df1, df2; columns = :union) ≅ DataFrame([[1, 2, 3, 7, 8, 9], 
+                                                             [4, 5, 6, missing, missing, missing]], 
```

Comment:
Can you also test the element type of resulting columns?

Also, please cut lines at 92 chars. The `A=[...]` syntax is sligthly more readable IMHO.

Reply:
Added element type, made columns correct, and changed constructors.